src/app.js

```diff
@@ -1 +1,148 @@
+    return unorderedList;
+}
+
+async function fetchCountries() {
+    try {
+        // het request maken
+        const response = await axios.get ('https://restcountries.com/v2/all');
+        // console.log(response.data[0].languages[1].name);
+
+        countryList.innerHTML = `    
+         <li> Eén van de talen is: ${response.data[0].languages[1].name} </li>
+        `;
+
+
+        // Creëer een unordered list om de landen in te plaatsen
+        createULElementWithId ();
+
+        // Array aanmaken met landnamen
+
+        const country = response.data.map ((country) => {
+            return country;
+        });
+
+        // Landen sorteren op aantal inwoners
+        country.sort ((a, b) => {
+            return b.population - a.population;
+        });
+
+        // Lijst element aan document object model (DOM) verbinden
+        let countryList = document.getElementById ('countryList');
+
+        // Schrijf een aparte functie voor elk regionaam en deze regio de correcte kleur teruggeeft als string.
+        country.forEach ((country) => {
+            const continent = country.region;
+            const textColor = setContinentColor (continent);
+
+            // voor ieder land de vlag url op halen.
+            const obj = country.flags;
+            const flagUrl = Object.values (obj)[1];
+            console.log (flagUrl);
+
+            // De url van de vlag afbeelding weergeven door middel van img-tag en de naam die bij de vlag hoort weergeven.
+            const imgTag = document.createElement ('img');
+            imgTag.setAttribute ('src', flagUrl);
+            imgTag.setAttribute ('alt', country.name + 'Flag');
+            imgTag.setAttribute ('class', 'flagImages');
+
+            const newLine = document.createElement ('br');
+            console.log (imgTag);
+            // const newline = document.createElement('br);
+            // console.log(imgTag);
+
+            const listItem = document.createElement ('li');
+
+            listItem.appendChild (imgTag);
+
+            const countryName = document.createElement ('h3');
+            countryName.setAttribute ('class', textColor);
+            countryName.textContent = `${country.name}`;
+
+            const textNodePopulation = document.createTextNode (`Has a population of ${country.population}`);
+            listItem.appendChild (countryName);
+            listItem.appendChild (newLine);
+            //listItem.appendChild(newline);
+            listItem.appendChild (textNodePopulation);
+
+            countryList.appendChild (listItem);
+        });
+
+
+    } catch (e) {
+        // de errors afvangen
+        console.error (e);
+    }
+
+
+    // de errors communiceren in de UI
+    if (e.response.status === 500) {
+        errorMessage.textContent = "Er ging iets mis in de server";
+    } else if (e.response.status === 404) {
+        errorMessage.textContent = "Het verzoek is mislukt";
+    }
+
+
+    function setContinentColor(countryRegion) {
```

Comment:
als je al een retrurn statement doet is de break niet meer nodig, na het uitvoren van het return statement verlaat ie de functie/switch namelijk al